can_tools/scrapers/puppet.py

```diff
@@ -144,9 +166,16 @@ async def _post_download(self, tmpdirname: str) -> pd.DataFrame:
         fns = glob.glob(os.path.join(tmpdirname, "*.csv"))
         assert len(fns) == 1
         with open(fns[0], "rb") as f:
-            tsv = io.BytesIO(f.read().decode("UTF-16").encode("UTF-8"))
+            try:
+                # Downloaded data from Crosstab needs this
+                tsv = io.BytesIO(f.read().decode("UTF-16").encode("UTF-8"))
+            except Exception as e:
+                # Downloaded data from Data doesn't
+                f.seek(0)
+                tsv = io.BytesIO(f.read())
             df = pd.read_csv(tsv, sep="\t")
 
+        # TODO0: ZW: Should we be deleting the tempfile once we are done loading it?
```

Comment:
I believe it will be automatically cleaned up when the context manager's scope ends in the `_get` method

Reply:
I do not believe this is true, but I will test. I had a ton of tmp folders floating around while testing, but they may have all been failed attempts that never got the chance to cleanup automatically.

Reply:
@sglyon: you are correct, the file and folders are automatically cleaned up when the context manager's scope ends. Assuming that it does so successfully. If there are any errors or you stop debugging in the middle of it, it sticks around and that is what I was seeing. Removing my comment.